.coveralls.yml

```diff
@@ -0,0 +1,2 @@
+service_name: travis-pro
+repo_token: N9vvhSxNlyMHui2fmbcKK1ao22WOXv7rC
```

Comment:
If you're using an open source plan, you don't need to have this configuration

Reply:
Are you sure Travis will read my repo without a token?

Reply:
Yes it will. The repo token is only necessary for when you're using the paid version of TravisCI which I don't think you're using. You can make a commit removing `coveralls.yml` and see; if it affects the build, let me know and I'll be happy to help troubleshoot. :smile: